test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
```

Comment:
I think we do. This is the only way to be consistent and to me it looks better this way.

Here we have inner arguments pushed off to the last indentation (here, the zeroth column) because the `arguments-newline` is `'(t)`. Since the closing delimiter offset is 0 (which will soon be hardcoded), we go all the way to `bol` at the next line.

But it wouldn't be hard to make the pushing behaviour a setting if you want.